src/app/core/entity-components/entity-details/form/form.component.ts

```diff
@@ -67,6 +67,22 @@ export class FormComponent implements OnInitDynamicComponent, OnInit {
     this.enablePhotoUpload = this.childPhotoService.canSetImage();
   }
 
+  calculateAge(selectedDateOfBirth: Date) {
```

Comment:
Proposal: Move this function to `utils.ts` and also remove it from the `child.ts` file

Reply:
Yes, we've already thought about that. I'll try to implement your proposal.